simplq/src/components/ErrorHandler.jsx

```diff
@@ -8,6 +8,7 @@ import PageNotFound from './pages/PageNotFound';
 import { setErrorNotifOpen } from '../store/appSlice';
 import { store } from '../store'; // TODO: Use Hooks
 
+// ToDo: make functional
 export class ErrorBoundary extends React.Component {
   constructor(props) {
```

Comment:
Do you know if it's possible to get the errorBoundary to work if it's made functional? Granted, it doesn't work right now, but I haven't seen a functional implementation.

Reply:
I guess making separate component file might help. Will try next.

Reply:
@maaverik  after spending an hour trying separating components, came across a statement on react doc site:-
"Only class components can be error boundaries." 😑

Reply:
Ah. Well, that settles it. Now we just have to figure out why it still doesn't work.